rawfile/bd2fs.py

```diff
@@ -308,7 +327,7 @@ def CreateSnapshot(self, request: csi_pb2.CreateSnapshotRequest, context):
                     creation_time=Timestamp(
                         seconds=creation_time_ns // nano, nanos=creation_time_ns % nano
                     ),
-                    ready_to_use=True,
+                    ready_to_use=False,
```

Comment:
If we're using copy_on_write isn't the snapshot instance, in which can we can return snapshot as ready?

Reply:
We are delighting the entire process to task manager and we know if the worker has picked up the job or not

Reply:
Sorry didn't get you?

Reply:
Even using CoW, we are doing it in the background using the background worker, So we don't know if the task manager has picked up the job or not

Reply:
Yeah that's my doubt, do we need copy manager if we're using cow? 
Otherwise we could have a small wait with timeout in case it's cow?

Reply:
Done.